IA-2216 Django gettext deprecation warning

olethanh opened the pull request:
ugettext_lazy was an alias for gettext_lazy, it is deprecated in our version of Django and will be removed from the next, so in preparation of a future upgrade this PR correct the usage.

Related JIRA tickets : IA-2216

## Self proofreading checklist

- [x] Did I use eslint and black formatters
- [x] Is my code clear enough and well documented
- [NA] Are my typescript files well typed
- [NA] New translations have been added or updated if new strings have been introduced in the frontend
- [NA] My migrations file are included
- [NA] Are there enough tests
- [NA] Documentation has been included (for new feature)

## Changes

Just did a mass replace


## How to test

No change in behavior expected

madewulf commented on plugins/polio/js/src/constants/translations/fr.json:
```diff
@@ -187,6 +187,7 @@
     "iaso.polio.label.regional": "Régional",
     "iaso.polio.label.reportingDistricts": "Districts rapportés",
     "iaso.polio.label.responsible": "Responsable",
+    "iaso.polio.label.restoreWarning": "Etes-vous sûr(e) de vouloir restorer cette campagne?",
```
Restaurer, pas restorer.

olethanh committed:
IA-2216 Django gettext deprecation warning

ugettext_lazy was an alias for gettext_lazy, it is deprecated in our version of Django
and will be removed from the next, so in preparation of a future upgrade this PR correct the usage.

mathvdh reviewed (approved):
ok
